documentation/book/con-kafka-mirror-maker-authentication.adoc

```diff
@@ -58,7 +58,7 @@ It is possible to use it only if the broker listener, clients are connecting to,
 SCRAM-SHA-512 uses a username and password to authenticate.
```

Comment:
_Please reword this concept module as follows:_

### SCRAM-SHA-512 authentication

To configure Kafka Mirror Maker to use SCRAM-SHA-512 SASL authentication, set the `type` property to `scram-sha-512`. The broker listener to which clients are connecting must also be configured to use SCRAM-SHA-512 SASL authentication. This authentication mechanism requires a username and password.

* Specify the username in the `username` property.
* In the `password` property, specify a link to a `Secret` containing the password in the `passwordSecret` property.
* In the `secretName` property, specify the name of the `Secret` and the key under which the password is stored inside the `Secret` in the `password` property. 

#### Example of SCRAM-SHA-512 client authentication configuration

[No changes to the code sample are needed.]